tests/e2e/tests/critical-path/monitor/save-commands.e2e.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from 'fs';
+import * as os from 'os';
+import {acceptLicenseTermsAndAddDatabase, deleteDatabase} from '../../../helpers/database';
+import { MonitorPage, CliPage } from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+import { rte } from '../../../helpers/constants';
+
+const monitorPage = new MonitorPage();
+const cliPage = new CliPage();
+const tempDir = os.tmpdir();
+const downloadsDir = `${process.env.USERPROFILE}/Downloads`;
+
+fixture `Save commands`
+    .meta({ type: 'regression' })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabase(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+    })
+    .afterEach(async() => {
+        //Delete database
+        await deleteDatabase(ossStandaloneConfig.databaseName);
+    });
+test
+    .meta({ rte: rte.standalone })('Verify that user can see a tooltip and toggle that allows to save Profiler log or not in the Profiler', async t => {
```

Comment:
Please set the `meta` on the fixture level if all are the same

Reply:
done

Reply:
So do you use with both fixture meta 'regression' and this one?

Reply:
yes